charts/enterprise-metrics/templates/graphite-querier-dep.yaml

```diff
@@ -0,0 +1,152 @@
+{{- if .Values.graphite.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "enterprise-metrics.fullname" . }}-graphite-querier
+  labels:
+    app: {{ template "enterprise-metrics.name" . }}-graphite-querier
+    chart: {{ template "enterprise-metrics.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  annotations:
+    {{- toYaml .Values.graphite_querier.annotations | nindent 4 }}
+spec:
+  replicas: {{ .Values.graphite_querier.replicas }}
+  selector:
+    matchLabels:
+      app: {{ template "enterprise-metrics.name" . }}-graphite-querier
+      release: {{ .Release.Name }}
+  strategy:
+    {{- toYaml .Values.graphite_querier.strategy | nindent 4 }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "enterprise-metrics.name" . }}-graphite-querier
+        # The name label is important for cortex-mixin compatibility which expects certain names for services.
+        name: graphite-querier
+        gossip_ring_member: "true"
```

Comment:
I'm not sure what this label indicates, but afaik the graphite querier should not have any interactions with any ring because it's now using the remote_read api, or not?

Reply:
:+1:  Addressed in https://github.com/grafana/helm-charts/pull/872/commits/6e7f1720ca574756febfc540dede43ed202fe22e